Tensile/Tests/extended/convolution_config/conftest.py

```diff
@@ -1,9 +1,24 @@
 import pytest
 import Tensile.Tensile as Tensile
+from collections import namedtuple
 from YamlBuilder.YamlBuilder import YamlBuilder
+from YamlBuilder.YamlBuilder import Solutions
 
 args={}
 
+TestConfig=namedtuple("TestConfig", ["solution", "problem_func"])
+Runner=namedtuple("Runner", ["func", "solution"])
+TensileState=namedtuple("TensileState", ["args"])
+
+# this control the default solutions used for each test.
+solutions = ["src1", "src5_gsu", "asm3_pbd", "asm3_splitu"]
+#solutions = ["asm3_pbd"] # run a smaller set of tests
+
+# shortcuts for setting parameters in tests:
+pytest.defaultSizes = pytest.param((YamlBuilder.ProblemSizes, 1), id="default_sizes")
+pytest.resnetSizes  = pytest.param((YamlBuilder.ProblemSizesResNet,1),id="resnet")
+pytest.inceptionSizes  = pytest.param((YamlBuilder.ProblemSizesInception,1),id="inception")
```

Comment:
This is still risky - you're making the assumption that pytest doesn't (and won't in the future) have anything with these names, or use introspection to modify these variables.